moped-editor/src/views/projects/projectView/ProjectTimeline.js

```diff
@@ -132,9 +132,9 @@ const ProjectTimeline = ({ refetch: refetchSummary }) => {
   );
 
   /**
-   * Phase table lookup object formatted into the shape that <MaterialTable>
+   * Subphase table lookup object formatted into the shape that <MaterialTable>
```

Comment:
thanks for the level of attention to the difference between phase and subphase here.